Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Chip.h

```diff
@@ -146,6 +146,7 @@ class Chip : public TObject
     /// @param p Point to be added
     void InsertPoint(Point *p);
```

Comment:
Yes, let us slowly but steadily move from `TObjArray` and `TClonesArray` towards strongly typed containers. `std::vector` is not the only stl sequence container, but most commonly known. It's a variable size container. In contrast to that, `std::array` is a fixed size container. So, depending on the use case there are different options.